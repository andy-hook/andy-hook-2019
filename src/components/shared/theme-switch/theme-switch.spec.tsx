import React from "react"
import renderer from "react-test-renderer"

import ThemeSwitch from "./theme-switch"

describe("<ThemeSwitch />", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<ThemeSwitch />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
