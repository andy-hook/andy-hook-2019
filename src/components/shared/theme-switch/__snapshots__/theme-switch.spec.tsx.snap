// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ThemeSwitch /> renders correctly 1`] = `
<div
  className="sc-bdVaJa GyhLv"
/>
`;
